examples/react-client/fishjam-chat/src/components/Tile.tsx

```diff
@@ -13,7 +13,7 @@ type Props = {
 
 export function Tile({ videoTrack, audioTrack, name, id }: Props) {
   const isMuted = !audioTrack || audioTrack.metadata?.paused;
-  const isSpeaking = audioTrack?.vadStatus === "speech";
+  const isSpeaking = useVAD([id])[id];
```

Comment:
**nitpick:** This API looks a bit weird 🤔 Can it be just `useVAD(id)`?

Reply:
it could and we will do it once we come back from holidays